openapi/components/schemas/GatewayAccountConfig/PSiGate.yaml

```
@@ -0,0 +1,29 @@
+description: PSiGate gateway configuration.
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        properties:
+          username:
+            type: string
+            description: Username or store name.
+          password:
+            type: string
+            description: Password.
+            format: password
+            writeOnly: true
+          clientId:
+            type: string
+            description: Client ID.
+          clientApiKey:
+            type: string
+            description: Client API Key.
```

nitpick: capitalization 
```suggestion
            description: Client API key.
```